Expedition.py

```diff
@@ -71,6 +72,24 @@ def __init__(self, site, pageName, db, format = None):
     self.gratAdd = u" ".join(graticule)
     self.gratAddr = u",".join(graticule)
     isGlobalhash = pageNameParts[1] == "global"
+
+    if self.page.isRedirectPage():
+      self.text = u""
+      self.people = None
+      self.categories = []
+    else:
+      self.text = self.page.get()
+      self.people = identifyParticipants(self.text, self.page, getLinks = True)
+      self.categories = self.page.categories()
+
+    if isGlobalhash:
+      meetup_in_categories = [
+        category.title() for category in self.categories
+        if RE_MEETUP_IN_LAT_LON.match(category.title())
+      ]
+      if len(meetup_in_categories) > 0:
+        lat, lon = meetup_in_categories[0].split(" ")[2:4]
+        self.gratAddr = ",".join([lat, lon])
     
     if isGlobalhash:
```

Comment:
It would seem to me like this should be combined with the condition above, and probably put in a comment why we're keeping `self.lat` and `self.lon` set to `None`, because as I look at the code, I keep wanting to set them correctly, but that will cause `self.gratName` to no longer be set to `Globalhash`.

Reply:
That's right. I'll combine them and put the comment. I was pretty close to populating `lat` and `lon` attributes, but eventually realised it'd break the `gratName`.